src/compressed_tensors/compressors/model_compressor.py

```diff
@@ -55,6 +57,20 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 
+if TYPE_CHECKING:
+    # dummy type if not available from transformers
+    CompressedTensorsConfig = TypeVar("CompressedTensorsConfig")
+
+
+def isCompressedTensorsConfig(compression_config: Any) -> bool:
```

Comment:
Lets make this sake case `is_comp....`

Reply:
See: https://github.com/neuralmagic/compressed-tensors/pull/176